src/eip.rs

```diff
@@ -28,7 +30,8 @@ pub(crate) async fn allocate_address(
         .iter()
         .map(|(k, v)| Tag::builder().key(k).value(v).build())
         .collect();
-    tags.push(Tag::builder().key(POD_UID_TAG).value(pod_uid).build());
+    tags.push(Tag::builder().key(EIP_UID_TAG).value(eip_uid).build());
+    tags.push(Tag::builder().key(EIP_NAME_TAG).value(eip_name).build());
```

Comment:
Just occurred to me: do we have to worry about the pod namespace, or does the EIP operator only operate on the namespace it's installed in?

Reply:
(nevertheless, it would be useful to have the namespace information on the EIP AWS object so people can tell where an allocation comes from)

Reply:
It only operates on one namespace, but yes, I can add the namespace to the AWS tags. We should probably also add that to the filters, so it only gets the ones in its namespace.

I might add the namespace filtering later, after the initial migration, since it might prevent cleanup of EIPs from the old version that don't have K8S EIP objects.